bin/run_xrp.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+set -e
+
+BLOCKCHAIN=xrp docker-compose -f docker/docker-compose.yaml up --build
+docker-compose -f docker/docker-compose.yaml rm -f
```

Comment:
Did running ever work in your tests? I think the API is slightly different:
```
Error in exporter work loop:  RippledError: ledgerNotFound
    at RequestManager.handleResponse (/opt/node_modules/xrpl/dist/npm/client/RequestManager.js:70:27)
    at Connection.onMessage (/opt/node_modules/xrpl/dist/npm/client/connection.js:230:37)
    at WebSocket.<anonymous> (/opt/node_modules/xrpl/dist/npm/client/connection.js:255:53)
    at WebSocket.emit (node:events:527:28)
    at Receiver.receiverOnMessage (/opt/node_modules/xrpl/node_modules/ws/lib/websocket.js:1178:20)
    at Receiver.emit (node:events:527:28)
    at Receiver.dataMessage (/opt/node_modules/xrpl/node_modules/ws/lib/receiver.js:528:14)
    at /opt/node_modules/xrpl/node_modules/ws/lib/receiver.js:479:23
    at /opt/node_modules/xrpl/node_modules/ws/lib/permessage-deflate.js:308:9
    at /opt/node_modules/xrpl/node_modules/ws/lib/permessage-deflate.js:391:7
    at afterWrite (node:internal/streams/writable:498:5)
    at onwrite (node:internal/streams/writable:478:7)
    at Zlib.cb (node:internal/streams/transform:171:7)
    at Zlib.processCallback (node:zlib:611:8) {
  data: {
    error: 'lgrNotFound',
    error_code: 21,
    error_message: 'ledgerNotFound',
    id: 1,
    request: {
      command: 'ledger',
      expand: false,
      id: 1,
      ledger_index: 0,
      transactions: true
    },
    status: 'error',
    type: 'response'
  }
}

```

Reply:
I need to check out the initial block, that I've set. For my testing purposes, the initial block was set way higher